cit-api/Dockerfile.pipeline

```diff
@@ -11,6 +11,22 @@ RUN az storage blob download \
              --account-key $AZURE_STORAGE_ACCOUNT_KEY \
              --auth-mode key
 
+RUN az storage blob download \
```

Comment:
So all of this code I have deleted. Did you add it back in? If so we want to remove it